src/Core/Vipr.Core/CodeModel/Vocabularies/Capabilities/OdcmBooleanCapability.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Vipr.Core.CodeModel.Vocabularies.Capabilities
+{
+    public abstract class OdcmBooleanCapability : OdcmCapability
+    {
+        protected bool BooleanValue;
```

Comment:
I specifically named it as BooleanValue to capture the "Bool" attribute of the capability annotation in the edmx.
For example - 
```xml
<Annotation Term="Org.OData.Capabilities.V1.DeleteRestrictions">
            <Record>
              <PropertyValue Property="Deletable" Bool="false" />
              <PropertyValue Property="NonDeletableNavigationProperties">
                <Collection>
                  <NavigationPropertyPath>parentSection</NavigationPropertyPath>
                  <NavigationPropertyPath>parentNotebook</NavigationPropertyPath>
                </Collection>
              </PropertyValue>
            </Record>
          </Annotation>
```xml